### ISSUE TITLE
NullPointerException while using plugin

### ISSUE THREAD

**Issue by dima74** [swdotcom/swdc-intellij#22]
**Describe the bug**
Exception occurred while using IDE (I just code, and don't do anything specific to Code Time)

**ISSUE TYPE**
- Bug Report

**Steps to reproduce**

I just do usual coding (edit files, etc), so unfortunately I don't have steps for reproduction

**ACTUAL RESULTS**
```
java.lang.NullPointerException
	at com.softwareco.intellij.plugin.sessiondata.SessionDataManager.getSessionSummaryData(SessionDataManager.java:42)
	at com.softwareco.intellij.plugin.sessiondata.SessionDataManager.incrementSessionSummary(SessionDataManager.java:67)
	at com.softwareco.intellij.plugin.KeystrokeCount.updateAggregates(KeystrokeCount.java:247)
	at com.softwareco.intellij.plugin.KeystrokeCount.processKeystrokes(KeystrokeCount.java:203)
	at com.softwareco.intellij.plugin.SoftwareCoEventManager.initializeKeystrokeCount(SoftwareCoEventManager.java:213)
	at com.softwareco.intellij.plugin.SoftwareCoEventManager.initializeKeystrokeObjectGraph(SoftwareCoEventManager.java:182)
	at com.softwareco.intellij.plugin.SoftwareCoEventManager.lambda$handleChangeEvents$0(SoftwareCoEventManager.java:112)
	at com.intellij.openapi.application.impl.ApplicationImpl$1.run(ApplicationImpl.java:238)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
```

**Desktop (please complete the following information):**
 - OS: Linux
 - Plugin version: Code Time v1.0.5
 - IntelliJ info:
```
IntelliJ IDEA 2019.3.3 (Ultimate Edition)
Build #IU-193.6494.35, built on February 11, 2020
Runtime version: 11.0.6-internal+0-adhoc.dima.jbr11 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Linux 5.5.11-arch1-1
GC: ParNew, ConcurrentMarkSweep
Memory: 8125M
Cores: 8
Registry: ide.balloon.shadow.size=0
Non-Bundled Plugins: BashSupport, PsiViewer, com.jetbrains.ChooseRuntime, com.mnw.tabmover, com.softwareco.intellij.plugin, com.wakatime.intellij.plugin, indent-rainbow.indent-rainbow, intellij.haskell, org.jetbrains.kotlin, izhangzhihao.rainbow.brackets, org.jetbrains.idea.grammar, org.toml.lang, org.rust.lang
```

Thanks for Code Time!

**Comment by xavluiz** [comment]
Thank you for the bug report. I will look at this now.

**Comment by brettmstevens7** [comment]
Hi @dima74 - we've made a change to how JSON data is being parsed, which affects both this issue and the previous issue with the readme file. 

Can you retest and let us know if this issue persists for you?

**Comment by dima74** [comment]
AFAIR exception happens only once for me, so looks like it is fixed. Thank you!

**Comment by xavluiz** [comment]
My apologies for the late response but I reproduced the issue. I published "1.0.13" to the jetbrains marketplace which is currently waiting review. Thank you.
